src/common/src/array/stream_chunk.rs

```diff
@@ -85,6 +129,7 @@ impl Default for StreamChunk {
         Self {
             ops: Arc::new([]),
             data: DataChunk::new(vec![], 0),
+            meta: None,
```

Comment:
Will we use meta field after finish backfill cdc?

i.e. in downstream executors will it use `meta` field somehow?

If not maybe it's better to wrap it the other way around instead:

```rs
struct StreamChunkMeta {
  meta: ...,
  chunk: StreamChunk,
}
```